docs/TestCoverage.md

```diff
@@ -5799,6 +5799,674 @@ expect(node, inputs=[x], outputs=[y],
 <br/>
 
 # Model Test Coverage
-## To be filled.
+## bvlc_alexnet
+
+24 of 24 nodes in this model are covered (100.0%)
```

Comment:
Probably here we should say something like

`In bvlc_alexnet, there are 24 nodes.` Instead list 24 of 24, 100%, it doesn't make too much sense. :-)